src/modules/market/components/market-header-message/market-header-message.jsx

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import classNames from "classnames";
+import PropTypes from "prop-types";
+import { SmallClose, Info } from "modules/common/components/icons";
+import ReactTooltip from "react-tooltip";
+import TooltipStyles from "modules/common/less/tooltip.styles";
+import Styles from "modules/market/components/market-header-message/market-header-message.styles";
+
+const MarketHeaderMessage = ({ marketId, dismiss, show, loggedIn }) => (
+  <>
+    {show && (
+      <div className={Styles.MarketHeaderMessage}>
+        <div style={{ display: "inline-block" }}>
+          <span className={Styles.MarketHeaderMessage__bolding}>
+            Always make sure that the{" "}
+            <span className={Styles.bolden}>title</span>,{" "}
+            <span className={Styles.bolden}>details</span>,{" "}
+            <span className={Styles.bolden}>reporting start time</span>,{" "}
+            <span className={Styles.bolden}>resolution source</span> and{" "}
+            <span className={Styles.bolden}>outcomes</span> are not in direct
+            conflict with each other{" "}
+            <span
+              className={classNames(
+                TooltipStyles.TooltipHint,
+                Styles.MarketHeaderMessage__tooltip__location
+              )}
+              data-tip
+              data-for="tooltip--message"
+            >
+              {Info}
+            </span>
+            <ReactTooltip
+              id="tooltip--message"
+              className={classNames(
+                TooltipStyles.Tooltip,
+                TooltipStyles.WideTooltip
+              )}
+              effect="solid"
+              place="top"
+              type="light"
+            >
+              <div
+                style={{ color: "#372e4b" }}
+                className={Styles.MarketHeaderMessage__tooltip}
+              >
+                {
+                  <div>
+                    <p>
+                      Markets on Augur are created by the community, this means
+                      that errors are can be made in the creation of a market,
```

Comment:
"are" and "can" should not both be here. get rid of "are"

Reply:
`  that errors can be made in the creation of a market,` this does read better. good one